src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * Class for printing UI elements.
+ */
+public class Ui {
+    private static final String MESSAGE_INTRO = "Hello! I'm Dusk\n     What can I do for you?";
```

Comment:
Is Dusk a typo? 🤣

Reply:
hahah nope I wanted to change the name to be more creative but many of the classes were still in the old `Duke` name